Makefile

```diff
@@ -96,6 +96,11 @@ test:
 test-examples:
 	python -m pytest -n auto --dist=loadfile -s -v ./examples/pytorch/
 
+# Run benchmark
+
+benchmark:
+	python3 benchmark/benchmark.py --config-dir benchmark/config --config-name generation --commit=diff backend.model=google/gemma-2b --multirun
```

Comment:
should detect the devices to run / run on all devices with a || ? 
Also we should run n this make bench static, dynamic and etc